src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -963,7 +964,7 @@ private MethodSymbol GetGetEnumeratorMethod(NamedTypeSymbol collectionType, Diag
             return getEnumeratorMethod;
         }
 
-        private ForEachEnumeratorInfo.Builder GetDefaultEnumeratorInfo(ForEachEnumeratorInfo.Builder builder, DiagnosticBag diagnostics, TypeSymbol collectionExprType)
+        private ForEachEnumeratorInfo.Builder GetDefaultEnumeratorInfo(ForEachEnumeratorInfo.Builder builder, DiagnosticBag diagnostics, TypeSymbol collectionExprType, bool isAsync)
```

Comment:
>, bool isAsync [](start = 166, length = 14)

It looks like this parameter is not used.

Reply:
Removed.